docs/source/prescription.rst

```diff
@@ -1615,3 +1610,47 @@ Suggested changes to the manifest files used for deployment.
 
 
 See :ref:`manifest_changes` section for more info and semantics.
+
+GitHubReleaseNotesWrap
+======================
+
+A specific type of wrap pipeline unit that adds links to GitHub release page.
+
+GitHubReleaseNotesWrap ``run.release_notes``
+############################################
+
+A list of entries describing release for which resolved a link
+a link to GitHub release should be constructed.
```

Comment:
`a link`

Reply:
Adjusted in d292f95b, thanks for spotting this 👍🏻